### PR TITLE
Add dynamic longevity score calculation

### LINKED ISSUE #105: Risk Framework: Calculate Longevity Score dynamically
To avoid setting the longevity manually, calculate it automatically based on the strategy activation value.

We should have a function to get the score based on days or months (from activation and now).

### PR DESCRIPTION
**Pull request by salazarguille**
It includes:

- Changes to calculating the longevity score dynamically based on the min activation date/time (strategies).

### CODE REVIEW

File: src/utils/commonUtils.ts [hunk 1]
```diff
@@ -185,6 +185,29 @@ export const getTvlImpact = (tvl: number): number => {
     return 5;
 };
 
+export const getLongevityScore = (days: number): number => {
```
Comment (storming0x) [inline src/utils/commonUtils.ts]: should we make a risk util.ts? founds like we will keep adding more and more to commons that may be fitting for a /utils/score or /utils/risk file

**Review by storming0x — CHANGES_REQUESTED** [review]
minor refactoring rest LGTM!

File: src/utils/commonUtils.ts (hunk 1 above)
Comment (salazarguille) [reply src/utils/commonUtils.ts]: Done. I created utils/risk file

**Review by storming0x — APPROVED** [review]
LGTM!

**Pull request merged**